server/dirstore.go

```diff
@@ -686,18 +686,15 @@ func (store *DirJWTStore) startExpiring(reCheck time.Duration, limit int64, evic
 			now := time.Now()
 			store.Lock()
 			if pq.Len() > 0 {
-				if it := heap.Pop(pq).(*jwtItem); it.expiration <= now.Unix() {
+				if it := pq.heap[0]; it.expiration <= now.Unix() {
```

Comment:
So you have second resolution only here, which I understand is fine based on usage. For tests with higher time based resolutions sometimes we do stuff like.

```go
now := time.Now().Sub(900*time.Millisecond)
```

This allows the test to run in ~100ms versus over a second. Just food for thought.

Reply:
unfortunately I am stuck with second resolution due to jwt

Reply:
Understood, but you can still use the trick above to speed up the test.

Reply:
But this is actual code, not test code. Are you saying that we do that in server code in some places? I don't think so.

Reply:
in test code we sometimes change expiration time of a jwt, or some other things. we can sometimes access server internals and change values, but this is done in test code itself, not in server code.

Reply:
are you suggesting to round down the current time? Is this done in other places of the code?

Reply:
No apologies, thought it was test code stuff.